tests/kernel/sleep/src/main.c

```diff
@@ -20,8 +20,9 @@
 #define TEST_THREAD_PRIORITY    -4
 #define HELPER_THREAD_PRIORITY  -10
 
-#define ONE_SECOND  (MSEC_PER_SEC)
-#define TICKS_PER_MS  (MSEC_PER_SEC / CONFIG_SYS_CLOCK_TICKS_PER_SEC)
+#define ONE_SECOND		(MSEC_PER_SEC)
```

Comment:
Nothing to do with this patch, but pet peeve: MSEC_PER_SEC is just silly to begin with.  SI number prefixes are an international standard that has been in regular use for over two hundred years.  Defining this as a constant is like defining "SKY_COLOR" to "BLUE".  It's what "milli" means.  We use preprocessor constants to abstract meaning that isn't obvious to the reader, not to move around the musical chairs of linguistic synonums.  And here this test isn't content to leave that alone, it has to do it twice!